keyboards/polly40/readme.md

```diff
@@ -0,0 +1,15 @@
+# 42
+
+42-keys ergonomic keyboard running on Adafruit Feather BLE.
+
+![42](https://github.com/nglgzz/42/raw/master/42.jpg)
+
+Keyboard Maintainer: [Angelo Gazzola (nglgzz)](https://github.com/nglgzz)  
+Hardware Supported: 42 (rev2, rev3), Feather 32u4 BLE  
+Hardware Availability: [GitHub](https://github.com/nglgzz/42/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```